### PR TITLE
Uses ensurepip to install pip rather than get-pip.py

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

File: linux/userdata_common.sh [hunk 1]
```diff
@@ -183,23 +183,32 @@ catch() {
...
 
-  # Install pip
-  stage="Install Python/Git" \
-    && curl "$PIP_URL" | python3 - --index-url="$PYPI_URL"
-  write-tfi "$stage" $?
+  install-pip
 
-  # Upgrade pip and setuptools
-  stage="Upgrade pip/setuptools" \
-    && pip install --index-url="$PYPI_URL" --upgrade pip setuptools
```
Comment (YakDriver) [inline linux/userdata_common.sh]: I don't know about not upgrading pip. I leaves us back at 9.0.1.
```
You are using pip version 9.0.1, however version 19.0.3 is available.
```

**Comment by YakDriver** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Pull request merged**